### DISCUSSION TITLE
Please, add a compact mode

### DISCUSSION THREAD

**Discussion by TsukiZero** [uBlockOrigin/uBlock-issues#2142] [Ideas]
The Classic UI was the closest we had to a compact UI. Here it is for reference:
![image](https://user-images.githubusercontent.com/78157224/173960155-454978bf-34f4-4380-a1a7-6c6c94302e90.png)

There was no actual reason to scrap this entirely, because the newer larger buttons in Fenix and the hostname info makes the popup anything between 5% and 50% larger than needed (specially if the hostname is longer than the right-hand pane, on top of the light shade of gray in dark mode becoming rather obnoxious.

I would be willing to use uiStyles to rectify that, but sadly that is not very viable due to the lack of documentation on the parts of the UI and the difficulty in pinpoint or customizing elements using View Source as the only available reference makes.

Speaking of uiStyles, may I suggest separating it from Advanced Settings so customization can be better organized and having some autocomplete? By this I mean: take Firefox's Developer Tools → Style Editor as an example: elements in the CSS there are color coded, for our convenience, if we try to edit anything there we have the auto-completion drop down to make our day easier.
Of course I don't mean to say Custom UI Style should be always on, it will require Advanced Settings to be turned on first and then Custom UI Style to be turned on separately and edited by clicking the gears next to it, similarly to how it is done to Advanced Settings.

**Comment by ghost** [comment]
Did you tried https://github.com/gorhill/uBlock/wiki/Advanced-settings#popupfontsize ? Seems to look promising on my side.

![image](https://user-images.githubusercontent.com/886325/174010598-de5430c0-9f64-4faa-9fa6-ac3230e207df.png)

![image](https://user-images.githubusercontent.com/886325/174010659-9d3a9082-831c-45bf-ad2c-0b0c02803e06.png)

**Reply by TsukiZero** [reply]
Unfortunately, not an ideal solution without proper styling.
